### PR TITLE
Fix/3rd party login

### PR DESCRIPTION
**Pull request by NatLee**
開發使用者註冊管理的時候，發現第三方登入的API有點問題

藉此機會修正一下

並且將`SOCIAL_GOOGLER_CLIENT_ID`拉出去作為環境變數

登入畫面修改為這樣

image

> 如果沒有填寫SOCIAL_GOOGLER_CLIENT_ID的話，將不會顯示Google登入的圖示

### CODE REVIEW

File: src/custom_auth/views.py [hunk 1]
```diff
@@ -1,51 +1,17 @@
-from django.contrib.auth import login
```
Comment (h-alice) [inline src/custom_auth/views.py:1]: Wow, that's a lot of deletion. 😲

File: src/login/templates/login-base.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<!doctype html>
```
Comment (h-alice) [inline src/login/templates/login-base.html:1]: Don't you hate writing front-end? : D

File: src/login/templates/login.html [hunk 3]
```diff
@@ -1,170 +1,156 @@
-<!doctype html>
```
Comment (h-alice) [inline src/login/templates/login.html:1]: Front-end codes are so redundant. 😟

**Review by h-alice — APPROVED** [review]
This is a quick code review since there's not much change to main program logic.

Everything is ok and ready to be merged.

File: src/custom_auth/views.py (hunk 1 above)
Comment (NatLee) [reply src/custom_auth/views.py:1]: Some redundant codes need to be removed ;)

File: src/login/templates/login-base.html (hunk 2 above)
Comment (NatLee) [reply src/login/templates/login-base.html:1]: GPT now is my front-end guy. :'(

File: src/login/templates/login.html (hunk 3 above)
Comment (NatLee) [reply src/login/templates/login.html:1]: Fucking CSS code :(

**Pull request merged**